package.json

```diff
@@ -33,6 +33,7 @@
     "form-data": "2.3.1",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
+    "jquery": "3.2.1",
```

Comment:
Why would you need jquery here? can we please remove it

Reply:
Sorry, fixed, was part of an afternoon of a lot of tries with different approaches.